framework_tvb/tvb/config/init/initializer.py

```diff
@@ -188,6 +188,10 @@ def _populate_algorithms(self, algo_category_class, algo_category_id):
                     stored_adapter.id = inst_from_db.id
 
                 stored_adapter = dao.store_entity(stored_adapter, inst_from_db is not None)
+
+                model = adapter_form().get_view_model()()
```

Comment:
`get_view_model` is a static method. You don't need to create instances for `adapter_form`, nor the returned view_model. This would cause problems if the `adapter_form` required some arguments (eg: `LocalConnectivityCreatorForm`)